releasenotes/notes/fix-aws-lambda-crashing-when-timeout-has-no-root-span-4a0fdacf707709a9.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    aws_lambda: This fix resolves an issue where the runtime would crash when no root span is found on an impending timeout. The root span provided by `datadog-lambda` is not recommended to be finished by the customer. But it will log a warning stating that no root span was found.
```

Can we simplify this? Would this work?
```suggestion
    aws_lambda: Resolves an unhandled exception which occurs when no root span is found before a lambda timeouts.
```